delivery_postlogistics/models/stock_picking.py

```diff
@@ -236,3 +236,8 @@ def generate_postlogistics_shipping_labels(self, package_ids=None):
         """Add label generation for PostLogistics"""
         self.ensure_one()
         return self._generate_postlogistics_label(package_ids=package_ids)
+
+    def action_generate_carrier_label(self):
+        for pick in self:
```

Comment:
for the sake of factorization, why not call `postlogistics_send_shipping` here?

Reply:
hello @nilshamerlinck, this PR has been updated.